### PR TITLE
pretify typscript code

### PR DESCRIPTION
**Pull request by s3341458**
just prettify everything in one go, in case I brother @DanielCollins by small pieces.

### CODE REVIEW

**Comment by s3341458** [comment]
currently there are actually syntax issues, due to #253 they are not shown, this pr actually helps reduce some of them.

However there are still syntax issues in the code base

File: typescript/src/entities/assignment.ts [hunk 1]
```diff
@@ -41,12 +41,12 @@ export class Assignment extends Entity {
   @Assignment.property()
   public bid?: Bid | null;
 
-  @Assignment.property({arrayType: "ProductionComment"})
+  @Assignment.property({ arrayType: 'ProductionComment' })
```
Comment (DanielCollins) [inline typescript/src/entities/assignment.ts]: why do you want an extra space after your braces?

**Comment by s3341458** [comment]
seems pretifier likes this.

https://eslint.org/docs/rules/object-curly-spacing it is a optional rule
for eslint

**Comment by s3341458** [comment]
while, I may need to take another look at this, but it is basically reasonable though I do not know why my syntax checker will do this.

I say it is a reasonable to have extra space in curely bracket because extra space in curely bracket in "import statement", according to the rule it is reasonable for eslint to apply this format to other parts as well.

BTW I configure my syntax fixer in a way, "prettifier" goes first then above that apply "eslint --fix"

**Comment by s3341458** [comment]
ok find my syntax fixer did it based on "eslint:recommended"

**Comment by DanielCollins** [comment]
note this one: https://github.com/merchisdk/sdk/issues/253#issuecomment-554761854

also,

> ok find my syntax fixer did it based on "eslint:recommended"

make sure you are using the included `.eslintrc.js ` (which does extend :recommended)

**Comment by DanielCollins** [comment]
the extra spaces are so ugly

**Comment by s3341458** [comment]
https://github.com/merchisdk/sdk/blob/7839c56bd3f92682a6f7f8d0b0e9e4024b77bda0/typescript/.eslintrc.js#L9

**Comment by s3341458** [comment]
@DanielCollins  it is not my choice why it is like this, I feel confused as well, the above result comes from following command on my side.

> ./node_modules/eslint/bin/eslint.js --fix 'src/*/**.ts'

seems like .eslintrc.js need to be adjusted

**Comment by DanielCollins** [comment]
> that is the point it is already in there

but, i don't reproduce the need for the extra spaces on my version.

**Comment by s3341458** [comment]
@DanielCollins has another  pr regarding related issue

**Pull request closed without merging**